sky/shell/platform/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -175,6 +175,75 @@ - (NSInteger)indexOfAccessibilityElement:(id)element {
   return (_children == nil) ? NSNotFound : [_children indexOfObject:element];
 }
 
+#pragma mark - UIAccessibilityAction overrides
+
+- (BOOL)accessibilityActivate {
+  // TODO(tvolkert): Implement
+  return NO;
+}
+
+- (void)accessibilityIncrement {
+  // TODO(tvolkert): Implement
+}
+
+- (void)accessibilityDecrement {
+  // TODO(tvolkert): Implement
+}
+
+- (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
+  BOOL canBeScrolled;
+  switch (direction) {
+    case UIAccessibilityScrollDirectionRight:
+    case UIAccessibilityScrollDirectionLeft:
+      canBeScrolled = _flags->canBeScrolledHorizontally;
+      break;
+    case UIAccessibilityScrollDirectionUp:
+    case UIAccessibilityScrollDirectionDown:
+      canBeScrolled = _flags->canBeScrolledVertically;
+      break;
+    default:
+      // Note: page turning of reading content is not currently supported
+      // (UIAccessibilityScrollDirectionNext,
+      //  UIAccessibilityScrollDirectionPrevious)
+      canBeScrolled = NO;
+  }
+
+  if (!canBeScrolled) {
+    return NO;
+  }
+
+  switch (direction) {
+    case UIAccessibilityScrollDirectionRight:
+      _bridge->server()->ScrollRight(_uid);
+      break;
+    case UIAccessibilityScrollDirectionLeft:
+      _bridge->server()->ScrollLeft(_uid);
+      break;
+    case UIAccessibilityScrollDirectionUp:
+      _bridge->server()->ScrollDown(_uid);
+      break;
+    case UIAccessibilityScrollDirectionDown:
+      _bridge->server()->ScrollUp(_uid);
+      break;
+    default:
+      DCHECK(false) << "Unsupported scroll direction: " << direction;
+  }
+
+  // TODO(tvolkert): provide meaningful string (e.g. page 2 of 5)
+  UIAccessibilityPostNotification(UIAccessibilityPageScrolledNotification, @"");
```

Comment:
From Docs: "When an assistive technology repeatedly receives the same scroll position string, it indicates to users that a border or boundary has been reached."

Does this mean we want to disambiguate the string instead of it just being empty. If we cannot provide anything useful, can we just say nil instead?